tests/e2e/support/index.js

```diff
@@ -23,6 +23,14 @@ Cypress.on('uncaught:exception', (err) => {
   }
 });
 
+// Hide Vue DevTools panel during tests to prevent it from covering UI elements
+Cypress.on('window:before:load', (win) => {
+  win.document.head.insertAdjacentHTML(
+    'beforeend',
+    '<style>.vue-devtools__panel { display: none !important; }</style>'
+  );
+});
```

Comment:
Interesting move. I actually like having the devtools open in the Cypress GUI so that I can troubleshoot with cypress + the devtools. I'd think we could either do this programmatically based on whether Cypress is in GUI mode, or otherwise we could configure it with a build time flag? But that introduces complexity in the build process. What do you think?

Reply:
So is the update to cypress 15 what causes the change in test behavior?